main.ts

```diff
@@ -64,12 +62,17 @@ export default class LetterboxdPlugin extends Plugin {
 					.then(async res => {
 						const parser = new XMLParser();
 						let jObj = parser.parse(res);
+						//console.log(jObj)
 						const filename = normalizePath('/Letterboxd Diary.md')
 						const diaryMdArr = (jObj.rss.channel.item as RSSEntry[])
-								.sort((a, b) => a.pubDate.localeCompare(b.pubDate)) // Sort by date
+								//.sort((a, b) => a.pubDate.localeCompare(b.pubDate)) // Sort by date
```

Comment:
I understand that sorting may not be working correctly but if it isn't being fixed here we should just leave this in